### PR TITLE
Create specific chat page

### PR DESCRIPTION
**Pull request by R303Vermeulen**
#31

### CODE REVIEW

**Comment by apandey16** [comment]
Side note, you can do a ```git stash``` to stash changes and then pop those changes instead of committing. TBH what you did is fine its just something to be aware of

**Comment by R303Vermeulen** [comment]
I was trying that and getting weird behavior from VsCode, maybe I could've reloaded it and it would've been fine. May need help doing it next time

**Comment by apandey16** [comment]
Another thing, a bit more of a nitpick, but could you add a button for the dev screen to go directly here? Just so if changes are made the page isn't fully detached/lost

**Comment by apandey16** [comment]
For the image, is it possible to center the filler text in the textbox?

**Comment by R303Vermeulen** [comment]
IMG_8AD65A63DFAA-1

**Review by apandey16 — APPROVED** [review]

File: Eat_and_Greet/src/comps/SpecificChat/SpecificChatTextbox/index.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { StyleSheet } from 'react-native';
```
[the remark this answers is not in the document]
Comment (R303Vermeulen) [reply Eat_and_Greet/src/comps/SpecificChat/SpecificChatTextbox/index.styles.tsx:1]: Did it so I could overlay the send button. Definitely could have configured the pre-existing textbox, but it was easier to just create a new one.

**Pull request merged**